daprdocs/content/en/operations/support/support-release-policy.md

```diff
@@ -57,15 +59,30 @@ General guidance on upgrading can be found for [self hosted mode]({{<ref self-ho
 |                          |                 1.1.2 |                    1.2.2 |
 |                          |                 1.2.2 |                    1.3.1 |
 |                          |                 1.3.1 |                    1.4.1 |
+|                          |                 1.4.1 |                    1.4.2 |
```

Comment:
This line is not needed, there is no reason to target 1.4.2 we should only mention the latest hotfix (see how 1.2.2 is treated) same for other entries below